### PR TITLE
introduced neo4j package for version 3.1.2

### LINKED ISSUE #991: Request to Bump Neo4j Version
Hi,

Neo4j has released their latest 3.1.1 version on [docker hub](https://hub.docker.com/_/neo4j/), it may looks like a minor version change however it might has solved a bug I encountered to recently. So pls consider bumping Neo4j to 3.1.1. Thx.

### PR DESCRIPTION
**Pull request by unterstein**
did the following:

copied neo4j/0 to neo4j/1 and replaced 3.1.0 with 3.1.2
copied neo4j-replica/0 to neo4j/1 and replaced 3.1.0 with 3.1.2

Fixes #991

### CODE REVIEW

**Comment by joerg84** [comment]
LGTM

File: repo/packages/N/neo4j-replica/1/resource.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "https://s3.amazonaws.com/downloads.mesosphere.io/universe/assets/icon-service-neo4j-small.png",
+    "icon-medium": "https://s3.amazonaws.com/downloads.mesosphere.io/universe/assets/icon-service-neo4j-medium.png",
+    "icon-large": "https://s3.amazonaws.com/downloads.mesosphere.io/universe/assets/icon-service-neo4j-large.png"
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "dcos-neo4j": "neo4j/neo4j-dcos:1.0.0-3.1.1"
```
Comment (joerg84) [inline repo/packages/N/neo4j-replica/1/resource.json]: Does it make sense to have the same version schema for the docker image than for the package version above?

Comment (unterstein) [inline repo/packages/N/neo4j-replica/1/resource.json]: ah, in the docker image version `1.0.0` and `3.1.1` are swapped...

**Comment by ryadav88** [comment]
@unterstein does the example need to updated to reflect these changes? 
https://github.com/dcos/examples/tree/master/1.8/neo4j

**Comment by unterstein** [comment]
Hi @ryadav88,

sorry for the delayed answer. No, no documentation change needed, just a bugfix release. Updated to newer 3.1.2 bugfix release in the meantime. Just replaced 3.1.1 with 3.1.2, hope this is not a problem.

Thanks
Johannes

**Pull request merged**